### ISSUE TITLE
git status

### ISSUE THREAD

**Issue by muhssamy** [microsoft/semantic-link-labs#166]
i am using the following code and i am getting an error

```python
import sempy_labs as labs
workspace = 'ws_retail_sm'
labs.get_git_status(workspace)
``` 

and i am getting the following error also i tried different workspaces

```python
---------------------------------------------------------------------------
FabricHTTPException                       Traceback (most recent call last)
Cell In[64], line 1
----> 1 labs.get_git_status(workspace)

File ~/cluster-env/clonedenv/lib/python3.10/site-packages/sempy_labs/_git.py:136, in get_git_status(workspace)
    133 response = client.get(f"/v1/workspaces/{workspace_id}/git/status")
    135 if response not in [200, 202]:
--> 136     raise FabricHTTPException(response)
    138 result = lro(client, response).json()
    140 for v in result.get("value", []):

FabricHTTPException: 200 OK for url: https://api.fabric.microsoft.com//v1/workspaces/d8d0536f-eb48-45c7-b629-cd96cc5f6868/git/status
Error: {"workspaceHead":"66e0a6d733cc9695f33838d28e6c3194f7b63427","remoteCommitHash":"66e0a6d733cc9695f33838d28e6c3194f7b63427","changes":[]}
Headers: {'Cache-Control': 'no-store, must-revalidate, no-cache', 'Pragma': 'no-cache', 'Content-Length': '110', 'Content-Type': 'application/json; charset=utf-8', 'Content-Encoding': 'gzip', 'Strict-Transport-Security': 'max-age=31536000; includeSubDomains', 'X-Frame-Options': 'deny', 'X-Content-Type-Options': 'nosniff', 'RequestId': 'a5709c1e-d376-4d50-9e77-e82e8f2e1b6f', 'Access-Control-Expose-Headers': 'RequestId', 'request-redirected': 'true', 'home-cluster-uri': 'https://wabi-west-europe-redirect.analysis.windows.net/', 'Date': 'Mon, 23 Sep 2024 18:37:39 GMT'}

```

**Comment by m-kovalsky** [comment]
Thanks for reporting. Fix will be available in the next release.

**Comment by m-kovalsky** [comment]
these will both be fixed.

**Comment by m-kovalsky** [comment]
Fixed in [0.8.0](https://github.com/microsoft/semantic-link-labs/releases/tag/0.8.0)
